web/template/layout/search_filter.gohtml

```diff
@@ -32,9 +32,6 @@
                 <div class="moj-filter__options">
                     <form class="form" method="get" id="search-filters">
                         <input type="hidden" name="term" value="{{ .SearchTerm }}"/>
-                        <button class="govuk-button" data-module="govuk-button" type="submit">
-                            Apply filters
-                        </button>
```

Comment:
If we remove the button, we should do so with JavaScript so that the form is still usable if JS isn't available.